crates/subspace-networking/src/shared.rs

```diff
@@ -73,16 +74,23 @@ pub(crate) struct Shared {
     pub(crate) connected_peers_count: AtomicUsize,
     /// Sender end of the channel for sending commands to the swarm.
     pub(crate) command_sender: mpsc::Sender<Command>,
+    /// Parent node instance (if any) to keep alive.
+    pub(crate) _parent_node: Option<Node>,
```

Comment:
I would add here an explicit `#[allow(dead_code)]` directive instead of the underscore. Please, also add more informative comment above - it's unclear without debugging why do we need this field here.